packages/react/src/flex/docs/overview.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Flex
+description: A CSS flexbox version of the Box component.
+group: Foundations
+storybookPath: /story/foundations-flex--basic
+relatedComponents: ['box', 'stack']
+---
+
+The Flex component is a version of [Box](/components/box) which applies `display: flex`.
+
+```jsx live
+<Flex>A simple Flex</Flex>
+```
+
+## As prop
+
+By default, the Flex component renders a HTML `<div>` element. You can render it with a different HTML element tag by using the `as` prop.
+
+```jsx live
+<Flex as="section">A simple Flex rendered as a section element</Flex>
+```
+
+## Inline
+
+Use the `inline` prop to apply `display: inline-flex` to the element.
+
+```jsx live
```

Comment:
```suggestion
```jsx live showCode
```

Reply:
I might stop repeating myself. Can we put `showCode` on all live code blocks for these components.